scripts/build_executables.py

```diff
@@ -79,16 +79,28 @@ def run_cmd(args) -> bool:
     ]):
         raise RuntimeError(f"could not make code.json")
 
-    # build binaries
+    # build single precision binaries
     build_args = [
         "make-program", ":",
         "--appdir", path,
-        "--double",
         "-fc", "ifort",
         "-cc", cc,
         "--zip", f"{path}.zip",
     ]
     if keep:
         build_args.append("--keep")
     if not run_cmd(build_args):
-        raise RuntimeError("could not build binaries")
+        raise RuntimeError("could not build single precision binaries")
```

Comment:
maybe the should be `"could not build single precision binaries"` since some are double precision by default (mf6 and swtv4)